apps/database-new/README.md

```diff
@@ -0,0 +1,93 @@
+<a href="https://demo-nextjs-with-supabase.vercel.app/">
```

Comment:
Seems to be the default README from nextjs starter - let's rewrite, doesn't have to be much